functions_task1.js

```diff
@@ -0,0 +1,9 @@
+function multiple(numberFirst, numberSecond) {
```

Comment:
We can just name the arguments 'first' and 'second'. It is understandable from the context that these are numbers